CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ accidentally triggering the load of a previous DB version.**
 * #5362 Make copy fetcher more async
 * #5336 Use NameData and namestrcpy for names
 * #5317 Fix some incorrect memory handling
+* #5384 Fix Hierarchical CAgg chunk_interval_size
```

Comment:
Nit: In the other changelog entries, we have not used any abbreviations

```suggestion
* #5384 Fix hierarchical continuous aggregates chunk_interval_size
```

Reply:
Fixed!